src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -51,7 +55,7 @@ internal void OnMessageLogged(ILogMessage msg)
 
         public void Startup(ViewStartupParams viewStartupParams)
         {
-            // Do nothing for now
+            pmExtension = viewStartupParams.ExtensionManager.Extensions.OfType<PackageManagerExtension>().FirstOrDefault();
```

Comment:
I think we should write code that interacts with extensions defensively - this code might return `null` - then later null needs to be checked for when accessing this field.